db/db_impl/db_impl.h

```diff
@@ -399,11 +399,12 @@ class DBImpl : public DB {
   virtual void EnableManualCompaction() override;
   virtual void DisableManualCompaction() override;
 
-  Status ApplyReplicationLogRecord(
-      ReplicationLogRecord record, std::string replication_sequence,
-      CFOptionsFactory cf_options_factory,
-      bool allow_new_manifest_writes,
-      ApplyReplicationLogRecordInfo* info) override;
+  Status ApplyReplicationLogRecord(ReplicationLogRecord record,
```

Comment:
You might have to update the signature in db_test.cc as well: https://github.com/rockset/rocksdb-cloud/blob/master/db/db_test.cc#L3397

Reply:
Thanks